liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DevMojo.java

```diff
@@ -460,9 +462,32 @@ public boolean compile(File dir) {
                 log.error("Unable to compile", e);
                 return false;
             }
-
         }
-       
+        
+        @Override
+        public void restartDevMode(final ThreadPoolExecutor executor) {
+            // shutdown tests
+            executor.shutdown();
+            // stopping server
+            stopServer();
+            
+            log.info("Restarting liberty:dev mode");
+            ProcessBuilder processBuilder = new ProcessBuilder();
+            String processCommand = "mvn liberty:dev";
```

Comment:
We should also find a way to pass through all of the `@Parameter` variables from the current mojo, since those might not be default if the user had specified them from the command line or from the pom.

Reply:
Should this be all the parameters in DevMojo or all the parameters in DevMojo, StartDebugMojoSupport, BasicSupport, and AbstractLibertySupport?